python/pyhie/allspark/model/deepseek_v3.py

```
@@ -665,34 +771,87 @@ def _load_and_permute(name):
                             expert_tensors = [None] * len(torch_name)
                             for f in futures:
                                 expert_tensors[futures[f]] = f.result()
+                        # Load and stack scale_inv for FP8 experts
+                        if is_fp8:
+                            def _load_scale(name):
+                                sk = name + '_scale_inv'
+                                if sk in torch_weight:
+                                    s = (_load_direct(sk)
+                                         if _load_direct else torch_weight[sk]).cpu()
+                                    if s.dim() == 2:
+                                        s = torch.permute(s, (1, 0)).contiguous()
```

Flake8 (7.3.0)

[error] 'torch' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)